tests/Makefile

```diff
@@ -2,32 +2,43 @@
 
 dirs := unit regression integration end-to-end
 
+# Skip looking up files when building
+ifndef CK8S_TESTS_BUILD
+
 # Pure bats tests
-bats_suites = $(filter-out %.gen.bats,$(foreach dir, $(dirs), $(wildcard $(dir)/*.bats)))
+bats_suites = $(filter-out %.gen.bats,$(foreach dir, $(dirs), $(shell find $(dir) -type f -name *.bats)))
 
 # Cypress tests
 cypress_suites := $(foreach dir, $(dirs), $(wildcard $(dir)/*.cy.js))
 cypress_gen := $(cypress_suites:.cy.js=.gen.bats)
 
 # Template tests
-template_suites := $(foreach dir, $(dirs), $(wildcard $(dir)/*.bats.yaml))
+template_suites := $(foreach dir, $(dirs), $(shell find $(dir) -type f -name *.bats.yaml))
 template_gen := $(template_suites:.bats.yaml=.gen.bats)
 
-config_path := ${CK8S_CONFIG_PATH}
+endif
 
-container_image := elastisys-ck8s-apps-tests:main
+container_image := compliantkubernetes-apps/tests:main
```

Comment:
I think this only works because the "repository" will be `compliantkubernetes-apps`? I guess it's fine but it's a bit of a misuse, no?

Reply:
I'll change it to a dash instead of a slash, I keep forgetting that containerd and docker defaults to dockerhub for most things.